internal/driver/glfw/window_test.go

```diff
@@ -1370,7 +1370,7 @@ func TestWindow_SetPadded(t *testing.T) {
 	if hasNativeMenu() {
 		menuHeight = 0
 	} else {
-		menuHeight = widget.NewToolbar(widget.NewToolbarAction(theme.ContentCutIcon(), func() {})).MinSize().Height
```

Comment:
if the canvas text is empty, will it return a height different from 0?

Reply:
Yes, the height of text is defined by its parameters - the width is what changes based on content